source/reference/ejson/stringify.txt

```diff
@@ -0,0 +1,540 @@
+             - The character to use to indent each level.
+
+   * - ``options``
+     -  boolean
+     -  Optional
+     -  Additional configuration options
+      
+        .. list-table::
+           :header-rows: 1
+           :widths: 25 25 50
+
+           * - Option
+             - Default
+             - Meaning
+
+           * - ``legacy``
+             - ``false``
+             - Use the Extended JSON v1 specification for output.
+
+           * - ``relaxed``
+             - ``false``
+             - Enable relaxed mode for Extended JSON. Returns native 
+               JavaScript types instead of BSON types when possible.
+
+           * - ``useBigint64`` 
+             - ``false``
+             - Enable native bigint support.
+
+Behavior
+--------
+
+You can call the EJSON interface from inside an interactive ``mongosh``
+session or from the system command line using ``--eval``.
+
+Call the EJSON interface from an interactive session: 
+
+.. code-block:: javascript
+
+   EJSON.stringify( db.sales.find().toArray(), null, 2  )
+
+Call the EJSON interface from the system command line: 
+
+.. code-block:: shell
+
+   mongosh --eval "EJSON.stringify( db.sales.find().toArray(), null, 2  )"
+
+To control how documents are passed to EJSON, use one of the ``mongosh``
+:ref:`cursor method <doc-cursor-methods>` iterators. 
+
+.. list-table::
+   :header-rows: 1
+
+   * - Iterator
+     - Characteristics
+
+   * - :method:`.toArray() <cursor.toArray>`
+     - Blocking, buffers the entire result
+
+   * - :method:`.foreach <cursor.forEach()>`
+     - Non-blocking, prints documents one-by-one
+
+   * - :method:`.next() <cursor.next>`
+     -  Non-blocking, manually iterate over results
+
+Examples
+--------
+
+To try these examples, first create the ``sales`` collection:
+
+.. code-block:: javascript
+
+   db.sales.insertMany( [ 
+      { custId: 345, purchaseDate: ISODate("2023-07-04"), quantity: 4, cost: Decimal128("100.60"), },
+      { custId: 346, purchaseDate: ISODate("2023-07-12"), quantity: 3, cost: Decimal128("175.45"), },
+      { custId: 486, purchaseDate: ISODate("2023-08-01"), quantity: 9, cost: Decimal128("200.53"), },
+   ] )
+
+Change Output Spacing
+~~~~~~~~~~~~~~~~~~~~~
+
+To increase indentation between levels, set the ``spacing`` option.
+
+.. code-block:: javascript
+
+   EJSON.stringify( db.sales.findOne( { custId: 345 } ), null , 5 )
+
+``EJSON.stringify()`` indents each document level five spaces.
+
+.. code-block:: javascript
+
+   {
+        "_id": {
+             "$oid": "64da90c1175f5091debcab26"
+        },
+        "custId": 345,
+        "purchaseDate": {
+             "$date": "2023-07-04T00:00:00Z"
+        },
+        "quantity": 4,
+        "cost": {
+             "$numberDecimal": "100.60"
+        }
+   }
+
+To highlight the indentation between levels, use the ``spacing`` option
+to set a string of dots.
+
+.. code-block:: javascript
+
+   EJSON.stringify( db.sales.findOne( { custId: 345 } ), null , "..." )
```

Comment:
I’m not sure if this is a good example because the result isn’t valid JSON and cannot be parsed by anything.

A possible use case for using a string as the indentation argument could be passing `'\t'` in order to get tab-indented output, but really, I would consider just not documenting this (and linking to the [MDN docs for `JSON.stringify()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify) instead if we feel that this is important to have referenced somewhere).